plugins/depthCamera/src/DepthCameraDriver.cpp

```diff
@@ -120,23 +127,25 @@ bool GazeboYarpDepthCameraDriver::close()
     // Copy & paste like a Monkey
     if (this->m_updateImageFrame_Connection.get())
     {
-        m_imageCameraPtr->DisconnectNewImageFrame(this->m_updateImageFrame_Connection);
+        m_depthCameraPtr->DisconnectNewImageFrame(this->m_updateImageFrame_Connection);
         this->m_updateImageFrame_Connection = gazebo::event::ConnectionPtr();
     }
 
     if (this->m_updateRGBPointCloud_Connection.get())
     {
-        m_imageCameraPtr->DisconnectNewImageFrame(this->m_updateRGBPointCloud_Connection);
+        m_depthCameraPtr->DisconnectNewImageFrame(this->m_updateRGBPointCloud_Connection);
```

Comment:
Actually the usage is commented out, because Gazebo does not produce pointCloud yet, only rgb and depth.
The code is there in case in future update of Gazebo they'll implement this feature we are ready.